src/AccessibilityInsights.SharedUx/Highlighting/HollowHighlightDriver.cs

```diff
@@ -94,15 +97,12 @@ void SetElementInternal(A11yElement element)
 
                 if (this.IsEnabled)
                 {
-                    this.Highlighter.IsVisible = true;
                     var text = GetHighlightText(element);
                     this.Highlighter.SetText(text);
                 }
             }
-            else
-            {
-                this.Highlighter.IsVisible = false;
-            }
+
+            this.Highlighter.IsVisible = visible;
```

Comment:
In the original code, the guard on line 95 ensured that `IsVisible` was only set to true if `IsEnabled` was also true. We can retain that behavior with something like this:
```suggestion
            this.Highlighter.IsVisible = this.IsEnabled && visible;
```

Reply:
Thanks!